config/prod.exs

```diff
@@ -10,7 +10,8 @@ use Mix.Config
 # which you should run after static files are built and
 # before starting your production server.
 config :school_house, SchoolHouseWeb.Endpoint,
-  url: [host: "example.com", port: 80],
+  http: [port: {:system, "PORT"}],
+  url: [scheme: "https", host: "beta.elixirschool.com", port: 443]
```

Comment:
```suggestion
  url: [scheme: "https", host: "beta.elixirschool.com", port: 443],
```

Reply:
Round about way to get formatters changes onto a branch you don't have access to :tada: